test/CoreTest.cpp

```diff
@@ -1356,6 +1360,7 @@ TEST_CASE("unavailable_backend", "[core]")
 #endif
 
 #if openPMD_HAVE_MPI
+    MPI_Init(nullptr, nullptr);
```

Comment:
If we need MPI init here, then I have the fear we also need the logic in serial user code that is coded against an MPI-enabled openPMD-api, no?

Reply:
> we also need the logic in serial user code that is coded against an MPI-enabled openPMD-api

(Note: this situation is tested in every MPI-enabled CI run by `SerialIOTests`)

Reply:
On Slack: ok, so its an actual bug here because we pass `MPI_COMM_WORLD` 5 lines below :)

Reply:
No, sorry - MPI context handling for Catch2 should all go in `test/CatchRunner.cpp`
https://github.com/openPMD/openPMD-api/blob/dev/test/CatchRunner.cpp

Is the wrong runner used here?